### PR TITLE
Create ruleset: openscad.org

### PR DESCRIPTION
**Pull request by Rotzbua**

### CODE REVIEW

File: src/chrome/content/rules/openscad.org.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!--
+  OpenSCAD
+  Open source 3D design tool
+  
+  No https offer:
+  openscad.org - redirect to www.
+  forum.openscad.org - forum
+  files.openscad.org - downloads
+  
+  Last test date: 10.05.2018
```
Comment (J0WI) [inline src/chrome/content/rules/openscad.org.xml]: Please see also https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

File: src/chrome/content/rules/openscad.org.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!--
+	OpenSCAD
+	Open source 3D design tool
+
+	Refused:
+		openscad.org
```
Comment (J0WI) [inline src/chrome/content/rules/openscad.org.xml]: Can be redirected/secured to `www`

Comment (Rotzbua) [reply src/chrome/content/rules/openscad.org.xml]: I do not know how to redirect from `http://openscad.org` to `https://www.openscad.org` by a ruleset.

Comment (J0WI) [reply src/chrome/content/rules/openscad.org.xml]: Just add `<rule from="^http://openscad\.org/" to="https://www.openscad.org/"/>` before the `<rule from="^http:" to="https:" />` rule and add `openscad.org` as target.

**Comment by J0WI** [comment]
You forgot to add the target ;)

**Comment by Rotzbua** [comment]
Sorry :|
Now it should work. Rebase and squash done.

**Comment by J0WI** [comment]
LGTM, thanks!

**Pull request merged**